serverless/functions/serverless-functions-quickstart.adoc

```diff
@@ -0,0 +1,28 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-quickstart"]
+= Quick start guide
+:context: serverless-functions-quickstart
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: {FunctionsProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+[id="serverless-functions-quickstart-prerequisites"]
+== Prerequisites
+
+Before you can complete the steps in the following procedures, you must ensure that:
+
+* {ServerlessProductName} is installed on your cluster.
+* You have installed the xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[`oc` CLI] and xref:../../serverless/installing-kn.adoc#installing-kn[`kn` CLI].
+* You have installed Docker Container Engine or podman, and have access to a publicly available image registry.
++
```

Comment:
I'm unsure about podman. I was not able to make it work with podman.

Reply:
Same question. What was the problem?